src/modules/commander/commander_tests/state_machine_helper_test.cpp

```diff
@@ -507,7 +507,8 @@ bool StateMachineHelperTest::mainStateTransitionTest()
 		struct vehicle_status_flags_s current_status_flags = {};
 
 		current_commander_state.main_state = test->from_state;
-		current_vehicle_status.is_rotary_wing = test->condition_bits & MTT_ROTARY_WING;
+		current_vehicle_status.vehicle_type = test->condition_bits & MTT_ROTARY_WING ?
```

Comment:
Maybe add a bracket around `test->condition_bits & MTT_ROTARY_WING` to make it clearer to read.